### PR TITLE
Updated corp/Contracts.xml.aspx

### PR DESCRIPTION
**Pull request by Dragonrun1**

### CODE REVIEW

File: docs/xmlapi/corporation/corp_contracts.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+            </tr>
+            <tr>
+                <td>contractID</td>
+                <td><strong>long</strong></td>
+                <td>ID of a specific contract <em>(Optional)</em></td>
+            </tr>
+        </tbody>
+    </table>
+
+See [/char/Contracts.xml.aspx](https://github.com/ccpgames/eveonline-third-party-documentation/blob/master/docs/xmlapi/character/char_contracts.md) for details.
```
Comment (polarina) [inline docs/xmlapi/corporation/corp_contracts.md]: Can you make this a relative link, please?

Comment (Dragonrun1) [inline docs/xmlapi/corporation/corp_contracts.md]: Should it start at docs/ or somewhere later?

Comment (polarina) [inline docs/xmlapi/corporation/corp_contracts.md]: Take a look at other pages that do hyperlinking. For example https://github.com/ccpgames/eveonline-third-party-documentation/blob/master/docs/xmlapi/constants.md

**Comment by Dragonrun1** [comment]
Path updated.

**Comment by polarina** [comment]
Thanks! Cherry-picked your commits as 9f707b4179faa1f70e6972bff801772702f77134.

**Pull request closed without merging**